ui/admin/app/templates/scopes/scope/policies/index.hbs

```diff
@@ -0,0 +1,7 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
```

Comment:
None blocking comment: Will be awesome if we can add a dummy heder or something similar, so when testing this we can see content that tells us we are on the section.

Reply:
Good point -  added!